modules/rhcos-enabling-multipath-day-1-power.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bibm_power/installing-ibm-power.adoc
+// * installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc
+
+[id="rhcos-enabling-multipathday-1-power_{context}"]
+= Enabling multipathing with kernel arguments on {op-system}
+
+You can enable multipathing at installation time for nodes that were provisioned in {product-title} 4.9 or later. {op-system} supports multipathing on the primary disk, allowing stronger resilience to hardware failure to achieve higher host availability. Although it is often preferable to modify kernel arguments as a day-2 activity,
+you might want to add kernel arguments to all master or worker nodes during initial cluster
+installation via the machine config.
+
+To add kernel arguments to master or worker nodes, you can create a `MachineConfig` object
+and inject that object into the set of manifest files used by Ignition during
+cluster setup.
+
+.Procedure
+
+. Change to the directory that contains the installation program and generate the Kubernetes manifests for the cluster:
++
+[source,terminal]
+----
+$ ./openshift-install create manifests --dir=<installation_directory>
+----
+
+. Decide if you want to add kernel arguments to worker or master nodes.
+
+* Create a machine config file, such as `99-master-kargs-mpath.yaml`, that instructs the cluster to add the `master` label and that identifies the multipath kernel argument, for example:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: "master"
+  name: 99-master-kargs-mpath
+spec:
+  kernelArguments:
+    - 'rd.multipath=default'
+    - 'root=/dev/disk/by-label/dm-mpath-root'
+----
+
+. To enable multipathing post-installation on worker nodes:
+
+* Create a machine config file, such as `99-worker-kargs-mpath.yaml`, that instructs the cluster to add the `worker` label and that identifies the multipath kernel argument, for example:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: "worker"
+  name: 99-worker-kargs-mpath
+spec:
+  kernelArguments:
+    - 'rd.multipath=default'
+    - 'root=/dev/disk/by-label/dm-mpath-root'
+----
++
+You can change `master` to `worker` to add kernel arguments to worker nodes instead.
```

Comment:
I don't think this is needed. You just showed us how to add worker nodes. I would remove this sentence.